wled00/remote_codes.h

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
I would prefer this to be in the .cpp file as it will most likely only be used there.